pkg/karpenter/karpenter_test.go

```diff
@@ -59,5 +59,46 @@ var _ = Describe("Install", func() {
 					To(MatchError(ContainSubstring("failed to install Karpenter chart: nope")))
 			})
 		})
+
+		When("creating a namespace is disabled", func() {
+			BeforeEach(func() {
+				installerUnderTest = &Installer{
+					Options: Options{
+						HelmInstaller:        fakeHelmInstaller,
+						Namespace:            "karpenter",
+						ClusterName:          "test-cluster",
+						CreateServiceAccount: true,
+						ClusterEndpoint:      "https://endpoint.com",
+						Version:              "0.4.3",
+						CreateNamespace:      false,
+					},
+				}
+			})
+			It("will not create a namespace", func() {
+				Expect(installerUnderTest.Install(context.Background())).To(Succeed())
+				_, opts := fakeHelmInstaller.InstallChartArgsForCall(0)
+				Expect(opts.CreateNamespace).To(BeFalse())
+			})
+		})
+		When("creating a namespace is enabled", func() {
+			BeforeEach(func() {
+				installerUnderTest = &Installer{
+					Options: Options{
+						HelmInstaller:        fakeHelmInstaller,
+						Namespace:            "karpenter",
+						ClusterName:          "test-cluster",
+						CreateServiceAccount: true,
+						ClusterEndpoint:      "https://endpoint.com",
+						Version:              "0.4.3",
+						CreateNamespace:      true,
+					},
+				}
+			})
+			It("will not create a namespace", func() {
```

Comment:
```suggestion
			It("will the namespace", func() {
```

Reply:
Hm? I'm testing that if create namespace is `false` it should **not** create the namespace.

Reply:
opened a new comment

Reply:
Uh. What in the. I was looking at the wrong test. This day man... I see now what you changed. :/